src/modules/75-cf/components/ListingPageTemplate/ListingPageTemplate.tsx

```diff
@@ -71,6 +77,21 @@ const ListingPageTemplate: React.FC<ListingPageTemplateProps> = ({
         content={headerContent}
       />
 
+      {headerActions && (
```

Comment:
Might be an idea to delve a little deeper when showing this and only show when at least one of the optional attributes are present. It's technically possible to show the sub-header when no docs URL is provided
```suggestion
      {headerActions?.docs && (
```